specs/experimental/dispute-game-v2/dispute-game-v2.md

```diff
@@ -0,0 +1,519 @@
+allow for the new claim structure.
+
+The branch claim, encompassing all nodes within it, is a subgame root. Child subgames within it may now post an
+alternative version of the sub-tree that any of the nodes in the branch commits to. In current terms, the "left-most"
+child is now "most deep" child. As such, branch claim subgames are resolved depth-first, short circuiting at the first
+valid child.
+
+In the case of the branch claim subgames, if there is a subcontext attached to the leaf node, the subcontext determines
+their validity. The subcontext resolving blocks the resolution of the subgame that it is attached to, as described in
+the [Bisection Game](#bisection-game) section.
+
+### Branch Claim Incentives
+
+#### Branch Identity
+
+For the first branch claim posted by any actor, all participation in sub-games spawned off of the branch by the actor
+who created the branch are unbonded. That is, branch claims have identity. Other actors may join the "team" of the
+creator of the branch claim, but will require posting a bond of their own if they do not own the branch they are
+supporting. For example, let's look at a game where a second honest actor continues bisection:
+
+```mermaid
+flowchart TD
+    A[1] --> B[2]
+    A --> C[3]
+
+    B --> B1[4]
+    B --> B2[5]
+    B1 --> B11[8]
+    B1 --> B12[9]
+    B2 --> B21[10]
+    B2 --> B22[11]
+
+    C --> C1[6]
+    C --> C2[7]
+    C1 --> C11[12]
+    C1 --> C12[13]
+    C2 --> C21[14]
+    C2 --> C22[15]
+
+    style A fill:#A5FE91,color:#000;
+    style B fill:#A5FE91,color:#000;
+    style B1 fill:#A5FE91,color:#000;
+    style B11 fill:#A5FE91,color:#000;
+    style C fill:#FF2535,color:#000;
+    style C1 fill:#FF2535,color:#000;
+    style C11 fill:#FF2535,color:#000;
+    style C12 fill:#DBFFD3,color:#000;
+```
+
+In this case, the <span style="color: #DBFFD3">light green</span> player supports the
+<span style="color: #A5FE91">dark green</span> player's branch, and continues their work by creating a new branch
+claim off of the <span style="color: #FF2535">red</span> player's. In this game, the branch claims rooted at
+positions `1`, `3`, and `13` are all bonded.
+
+If the <span style="color: #A5FE91">dark green</span> player made the branch claim rooted at `13` first, they would
+not have had to post a new bond. If tree was deeper, and the <span style="color: #A5FE91">dark green</span> player
+supported the <span style="color: #DBFFD3">light green</span> player's branch claim rooted at position `13`, they
+would be able to re-use their bond for the branch claim rooted at position `1`.
+
+#### Bond Calculation
+
+With branch claims, the cost of the bond now scales with the _length of the branch_, rather than the
+_depth of the claim_. That is, branch claims made higher up in the tree cost more to make.
+
+The calculation of the bond is left to the implementor. It _must_ be a pure function of the definition $B(l)$, where
```

Comment:
We will want to specify the bond function in this document eventually

Reply:
Do we want to get that specific when it comes to the implementation details? The requirement for `l` being the scalar should be enough to hold the required invariant of the design. Further impl details can be up to the implementor of it.